### ISSUE TITLE
move certain elements out of the bottom left bar into the appbars at the top 

### ISSUE THREAD

**Issue by spydon** [ubuntu/app-center#1725]
@anasereijo 
If you might be concerned about the whitespace, we could also move certain elements out of the bottom left bar into the appbars at the top
![grafik](https://github.com/ubuntu/app-center/assets/15329494/59d11fa0-3c85-49da-8836-5b4fbee4658b)
(example mockup)
I think a fully fletched dedicated menu entry only for "about" is very uncommon anyways
what modern gnome apps often do, is place a hamburger menu in the top left bar if it is a paned application
which then leads to a setting dialog
![grafik](https://github.com/ubuntu/app-center/assets/15329494/77e6c2b0-b793-424e-9012-4188a6956b17)
![grafik](https://github.com/ubuntu/app-center/assets/15329494/20ac3e2c-d77f-43b7-b1ee-2748545d0058)
(examples and mockups)

_Originally posted by @Feichtmeier in https://github.com/ubuntu/app-center/issues/1621#issuecomment-2043277573_

**Comment by Feichtmeier** [comment]
Thanks @spydon 
I have become less certain about the settings hamburger menu, because maybe some would prefer to have this place for hide/unhide sidepanel, or we need two icons in the left bar, which would probably also be okay? 🤔

**Comment by spydon** [comment]
It would probably look pretty strange with two icons in the left bar? :thinking:

**Comment by Feichtmeier** [comment]
yes, I also have no super good answer on this one because we might need a solution for small screens, to hide the sidebar when the window becomes too narrow
as for example youtubemusic desktop does it
or my funny little weather app : D
<img width="640" alt="Bildschirmfoto 2024-07-01 um 17 21 45" src="https://github.com/ubuntu/app-center/assets/15329494/254170d9-6389-4ea3-b824-d766ca176473">
<img width="640" alt="image" src="https://github.com/ubuntu/app-center/assets/15329494/0ea31c79-5eea-4f26-b2e8-9a2f34c5a5ef">
